docs/zkapps/faq.mdx

```diff
@@ -1,36 +1,56 @@
 ---
-title: o1js and zkApps FAQ
+title: zkApps FAQ
 sidebar_label: FAQ
-description: Answers to common questions about zkApps and o1js, a library that creates zk circuits from user code. Learn about the use of built-in o1js datatypes, functions, and the implications for proof generation.
+description: Answers to common questions about zkApps (zero knowledge apps), the Mina Protocol smart contracts powered by zero knowledge proofs.
 keywords:
-  - o1js
   - FAQ
   - smart contract
   - zkApps
 ---
 
-## What files do I use to write zkApps?
+# zkApps FAQ
+
+Answers to common questions about zkApps (zero knowledge apps), the Mina Protocol smart contracts powered by zero knowledge proofs.
+
+### How do I stay up to date with zkApps?
+
+Follow the official O(1) Labs channels:
+
+- [Twitter/X @o1_labs](https://twitter.com/o1_labs)
+- [Blog](https://blog.o1labs.org/)
+
+### Where can I ask questions and contribute answers?
+
+[Mina Protocol Discord](https://discord.gg/minaprotocol) is the most popular place where Mina enthusiasts and technical contributors gather. 
+
+Join us in these zkApps channels:
+
+* [#zkapps-developers](https://discord.com/channels/484437221055922177/915745847692636181) to meet other developers building zkApps with o1js
+* [#zkapps-general](https://discord.com/channels/484437221055922177/910549624413102100) to ask general questions about zkApps, how to use a zkApp, and so on
+* [#zkapps-questions](https://discord.com/channels/484437221055922177/1047214314349658172) to ask zkApps-related questions and see Q&A history
```

Comment:
@barriebyron These channels are also relevant for o1js as it's the defacto place for developers to ask questions (not just about zkapps)

Reply:
per our convo we will combine into a single FAQ for zkApps and o1js